birthday/README.md

```diff
@@ -1,92 +1,90 @@
-Birthday CC V2.0
-================
+# Birthdays
+This custom command adds birthday functionality to your server, wishing members all the best on their respective date.
 
-# Info
+## Features
+* Send a configurable message in the chat on birthdays
+* Kick or ban users younger than 13 years
+* Allow users to set their birthday 
+* Allow staff to edit/remove birthdays
 
-| ℹ If you used V1.0 of this command, read `UPDATING FROM V1.0` at the bottom of this doc before updating. |
-| --- |
-<p>
-This command will send a congratulation message to the users of your server on their birthday.<p>
-This code can kick/ban users if they are under 13 years old, if you want it to.<p>
-Change <b>ONLY</b> the user variables.<p>
-User can only set their birthday once. After that a mod will have to set a new one or delete the existing one.<p>
+## Installing
+As usual, there are leading comments in each file describing where to put the script and which trigger to use. 
```

Comment:
There is only one file and no leading comments, the only comments are explaining the variables, all explanations are on this file.